maintainers/maintainer-list.nix

```diff
@@ -1447,6 +1447,11 @@
     github = "endgame";
     name = "Jack Kelly";
   };
+  ente = {
```

Comment:
I'd prefer if it was the same as your GitHub username. Then we wouldn't have to do the mapping, to look up your account.

Reply:
I did change that.